packages/material/scss/scrollview/_theme.scss

```diff
@@ -0,0 +1,34 @@
+@import "~@progress/kendo-theme-default/scss/scrollview/theme";
+
+@include exports("scrollview/theme/material") {
+    $scrollview-text-opacity: .4 !default;
+    $scrollview-padding-y: 10px !default;
+    $scrollview-padding-x: $padding-y !default;
+
+    .k-scrollview-next,
+    .k-scrollview-prev {
+        span {
+            color: $base-text;
+
+            &:hover {
+
+                &::before {
+                    background: rgba($base-bg, $scrollview-text-opacity);
+                }
+            }
+        }
+    }
+
+    .k-scrollview-pageable {
+        bottom: $scrollview-padding-y * 2;
+
+        li.k-button {
+            background: rgba(black, $scrollview-text-opacity);
+            margin: $scrollview-padding-y $scrollview-padding-x;
```

Comment:
Instead of setting the margins here, introduce these variables in the `theme-default` layout file, and override them before importing it in `_layout`. Renaming them to `$scrollview-pager-spacing-x` / `-y` seems more descriptive. This may also take care of the `bottom` property a few lines up, if it is calculated in theme-default.